pkg/sql/sem/tree/show.go

```diff
@@ -129,12 +131,179 @@ func (node *ShowBackup) Format(ctx *FmtCtx) {
 		ctx.WriteString(" IN ")
 		ctx.FormatNode(&node.InCollection)
 	}
-	if len(node.Options) > 0 {
+	if !node.Options.IsDefault() {
 		ctx.WriteString(" WITH ")
 		ctx.FormatNode(&node.Options)
 	}
 }
 
+type ShowBackupOptions struct {
+	AsJson               bool
+	CheckFiles           bool
+	DebugIDs             bool
+	IncrementalStorage   StringOrPlaceholderOptList
+	DecryptionKMSURI     StringOrPlaceholderOptList
+	EncryptionPassphrase Expr
```

Comment:
Just noting: We don't currently have an implementation of walkableStmt for ShowBackup. If we did, I would say we should probably be updating it to account for these new Exprs. We may want to add one in a follow up.

Reply:
Sounds good. Just added a todo. I don't know enough about this method to understand why one would need it. It seems that no other SHOW cmd in this file implements one either.